opm/input/eclipse/EclipseState/Grid/Carfin.hpp

```diff
@@ -94,7 +95,9 @@ namespace Opm
         int NX() const;
         int NY() const;
         int NZ() const;
-
+        size_t num_parent_cells() const;
+        std::tuple<std::vector<std::size_t>, std::vector<std::size_t>, std::vector<std::size_t>>
```

Comment:
put each std::vector entry on a separate (algined) line and put the function name directly below std::tuple